### ISSUE TITLE
insert new proposals when in voting phase

### ISSUE THREAD

**Issue by giovanisp** [fairdemocracy/vilfredo-client#208]
enable users to post new proposals when in voting phase

**Comment by pietrosperoni** [comment]
This should only be an option for questions where proposals do not dominate each other, and only have a single generation. So basically now how questions, but what questions.

**Comment by giovanisp** [comment]
Why not how questions after the first generation ?  Someone can make a proposal I like but with slightly different terms, then I'd like to copy it ("fork") and propose to others...

**Comment by pietrosperoni** [comment]
This has to do with the structure of the process. 

If we have several generations, than each generation has a fixed amount of time, and it is important that each proposal is equally evaluated. Also the generations delete some proposals and insert some others. If you let people shortcircuit this process the whole idea of generation becomes useless, but then you don't have anymore a process to push the system toward more integrated proposals. (Something which right now is done by extracting the pareto front).

On the other side the pareto front has the effect of deleting some proposals. And this is ok in how questions because at the end you are looking for 1 single answer. Not for multiple answers. So even when you have two good answers, you still need to chose one. On the other side in a what questions two good answers do not need to compete against each other.

I have been thinking a lot how to avoid the whole issue of generations in how questions. SO not having any generation at all, just let everybody write new proposal in any moment, and let any proposal be dominated in any moment. This "could" work in closed groups of people where you keep track of what proposals each user has seen, and then you make sure that people see every proposal. But it is much more complicated. 

SO basically the research is just not there to make sure that each proposal is equally evaluated AND the system moves toward consensus integrating every point of view. This is 3.0 or 4.0 stuff. Definitely Maybe SOmeday.
